### DISCUSSION TITLE
Hyprland troubleshooting help

### DISCUSSION THREAD

**Discussion by Qpernicus** [hyprwm/Hyprland#1958] [Q&A] [answered]
Soo.. I'm not sure how to tackle this. Since a few days my Hyprland session keeps crashing when i lock the screen (and it goes to black screen).. My config is here: https://codeberg.org/qpernicus/Dotfiles

I didnt really want to create an issue because i'm very uncertain if this is a Hyprland issue. 

So i could use a little help in finding the culprit.. Here is my train of thoughts so far:

I'm using SDDM-git and have had some issues there, where my sessions exits to a sddm login screen. So i thought, perhaps thats what causing it. So i disabled SDDM. But the problem persists. 

I then thought, i dabbled with my EWW config recently, perhaps i used a script with an error and thats causing a crash. So i disabled EWW so i could examine my config.. Nothing there.. And the problem persisted. Both with EWW or Waybar in use.

But the exact issue is that, when i lock my screen and/or swayidle activates, my session crashes after a while being in a black screen. and my laptop-fan goes haywire. So after reactivating SDDM i logged in and opened Btop and noticed my swaylock is using 100% CPU. I use this command to start swayidle:

`exec-once = swayidle timeout 600 "wlopm --off \*;swaylock -F -i ~/.cache/wallpaper --effect-blur 10x5 --clock --indicator" resume "wlopm --on \*"`

But i changed this to (mentioned on Hyprland Wiki) 

`exec-once = swayidle timeout 300 'swaylock -F -i ~/.cache/wallpaper --effect-blur 10x5 --clock --indicator' timeout 300 'hyprctl dispatch dpms off' resume 'hyprctl dispatch dpms on'
`
But that didn't fix the issue.. When i use SDDM it ' crashes into a loginscreen' , when i only use tty it goes there. The errors are a bit gibberish in the sense that there seem to be multiple lines shown on top of eachother,.. but i'm struggling to ' read' the cause of the crash. 

When i'm back on tty, it shows:

Could not connect to socket /run/seatd.sock: no such file or directory
Backend seatd failed to open seat. 

(amongst other things)

So i installed seatd which apparantly i hadn't previously.. But now the first error changes to No permissions.. which is odd. 

The rest of the 'errorpart'  reads:

Failed to flush wayland socket Broken pipe
And some other error about wayland i can't read because there is a Discord mention on top of it.. kinda.. 

But when i print the log (see attached) i dont see anything about a crash)
[Hyprlog.txt](https://github.com/hyprwm/Hyprland/files/11158886/Hyprlog.txt)

So i'm a bit lost if this is a Hyprland issue, or if i need to look elsewhere...

And feel free to respond with: "Create an Issue here"  and then close this.

**Comment by vaxerski** [comment, marked as answer]
making an issue would be easier.

Attach a crash report while you're at it

if you are not on -git, get on -git and check there.

**Comment by Qpernicus** [comment]
Created an issue: https://github.com/hyprwm/Hyprland/issues/1960
